.cirrus.yml

```diff
@@ -1,5 +1,96 @@
 env:
   CIRRUS_CLONE_DEPTH: 0 # Unshallow clone to obtain proper GIT_VERSION
+  BUILD_REPOSITORY_NAME: $CIRRUS_REPO_FULL_NAME
+  SYSTEM_PULLREQUEST_SOURCEBRANCH: $CIRRUS_BRANCH
+  SYSTEM_PULLREQUEST_PULLREQUESTID: $CIRRUS_PR
+  BUILD_SOURCEVERSION: $CIRRUS_CHANGE_IN_REPO
+  RPCS3_TOKEN: ENCRYPTED[13a0f18de9285e6c880e3aecbb54258245a74872af7d0fcb92447fa1200ceb6a4dc8acc880ddbf5e653d46336563ca72]
+
+windows_task:
+  matrix:
+    - name: Cirrus Windows
+      windows_container:
+        image: xddxd/windowsservercore:single
+        cpu: 8
+        memory: 16G
+  env:
+    CIRRUS_SHELL: "bash"
+    COMPILER: msvc
+    QT_VER_MAIN: '5'
+    BUILD_ARTIFACTSTAGINGDIRECTORY: 'C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\artifacts\'
+    QT_VER: '5.15.2'
+    QT_VER_MSVC: 'msvc2019'
+    QT_DATE: '202011130602'
+    QTDIR: C:\Qt\${QT_VER}\${QT_VER_MSVC}_64
+    VULKAN_VER: '1.2.154.1'
+    VULKAN_SDK_SHA: 'b64471f3a720e649c1fae6535ea83b8c642655ebed1485bfdf15bf4d88f746d9'
+    VULKAN_SDK: C:\VulkanSDK\${VULKAN_VER}
+    CACHE_DIR: "./cache"
+    UPLOAD_COMMIT_HASH: 7d09e3be30805911226241afbb14f8cdc2eb054e
+    UPLOAD_REPO_FULL_NAME: "rpcs3/rpcs3-binaries-win"
+  deps_cache:
+    folder: "./cache"
+  obj_cache:
+    folder: "./tmp"
+  obj2_cache:
+    folder: "./rpcs3/x64"
+  setup_script:
+    - './.ci/get_keys-windows.sh'
+    - './.ci/setup-windows.sh'
+#  spirv_script:
+#   - export PATH=${PATH}:"C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin"
+#   - cd "C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\Vulkan\spirv-tools-build"
+#   - msbuild.exe spirv-tools-build.vcxproj //p:Configuration=Release //m
+  rpcs3_script:
+#    - echo '".\.ci\build-cirrus-rpcs3.bat"' | cmd
+    - export PATH=${PATH}:"C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin"
+    - cd "C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build"
+    - msbuild.exe rpcs3.sln //p:Configuration=Release //m
+  deploy_script:
+    - mkdir artifacts
+    - source './.ci/export-cirrus-vars.sh'
+    - './.ci/deploy-windows.sh'
+  artifacts:
+    name: Artifact
+    path: "*.7z"
+#  push_script: |
```

Comment:
Instead of commenting this out, can we make this only toggle with the presence of the github token env var?